packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -156,6 +197,29 @@
   margin-inline-start: theme("margin.auto");
 }
 
+#close,
+#collapse,
+.back-button,
+calcite-action-menu {
+  --calcite-action-background-color: var(--calcite-panel-actions-background-color);
+  --calcite-action-background-color-hover: var(--calcite-panel-actions-background-color-hover);
+  --calcite-action-background-color-pressed: var(--calcite-panel-actions-background-color-pressed);
+  --calcite-action-text-color: var(--calcite-panel-actions-text-color);
+  --calcite-action-text-color-pressed: var(--calcite-panel-actions-text-color-pressed);
+}
+
+#close,
+#collapse,
+calcite-action-menu {
+  &:last-child {
+    --calcite-action-corner-radius-start-end: var(--calcite-panel-corner-radius);
```

Comment:
`--calcite-action-corner-radius-start-end` token will be removed per our conversation about handling border radius on buttons. Instead we can just target the style itself. 

```css
#close,
#collapse,
calcite-action-menu {
  &:last-child {
    border-start-end-radius: var(--calcite-panel-corner-radius, var(--calcite-corner-radius);
  }
}
```